bin/sem-init

```diff
@@ -1,24 +1,23 @@
 #!/usr/bin/env ruby
 # == Initializes an existing git repository to support schema evolutions managed
-#    by software-evolution-manager
+#    by schema-evolution-manager
 #
 # == Usage
-#  sem-init --dir <dir> --name <name> --user <username>
+#  sem-init --dir <dir> --url <url>
 #
 #   dir: The directory containing a git repository that will contain the schema evolution files
-#   name: The name of the actual database that will be managed
-#   user: The postgresql username to connect to this database
+#   url: The connection string for the psql database
 #
 # == Example
 #  git init /tmp/example_repo
-#  bin/sem-init --dir /tmp/example_repo --name example_us_development --user dev
+#  bin/sem-init --dir /tmp/example_repo --url postgresql://username@localhost/sample_development
 #
 
 load File.join(File.dirname(__FILE__), 'sem-config')
 SchemaEvolutionManager::Library.set_verbose(true)
 
-args = SchemaEvolutionManager::Args.from_stdin(:required => %w(name user dir))
-SchemaEvolutionManager::Preconditions.check_state(File.directory?(args.dir), "Dir[%s] does not exist" % [args.dir])
+args = SchemaEvolutionManager::Args.from_stdin(:required => %w(dir url))
+SchemaEvolutionManager::Preconditions.check_state(File.directory?(args.dir), "Dir[%s] does not exist" % args.dir)
```

Comment:
Ok with me, but it might come off as inconsistent to not support `user`, `host` here vs. `sem-apply`.